turbinia/workers/__init__.py

```diff
@@ -1080,22 +1102,33 @@ def run_wrapper(self, evidence):
         self.result = self.run(evidence, self.result)
 
       # pylint: disable=broad-except
+      except SoftTimeLimitExceeded as exception:
```

Comment:
As discussed offline

- we may revisit timeouts in the future to see if it's possible to use Celery timeouts and process termination exclusively. 
- we may want to catch this in the ```execute()``` method as well

Reply:
Yeah,  agree that we should look at only using Celery to manage this in the future and can hopefully integrate them in a way that would allow us to keep our metadata (redis) in sync.

I added the soft time limit exception to be caught in the `execute()` method as well and that seems to work well in testing.